[sarang_daddy] feature1 - 학습계획서 작성

sarangdaddy opened the pull request:
# 학습 방향 검토 중
# 필요 작업량이 방대하기에 작업 분류 고민 중

sarangdaddy committed:
First commit for only test

sarangdaddy committed:
학습계획서 작성

- 사전 학습키워드 정리
- 학습 계획 리스트 작성

crongro reviewed (commented):
벌써 공부하시는 내용이 보이는군요.
콘텐츠가 많으면 wiki나 notion 작성후에 링크를 다시는것도 좋겠어요.

sarangdaddy committed:
index.html 작성

- index html 최초 구성 생성
- 상단바(header bar) 구조 생성
- 상단바(header bar) 아이콘 및 이름 삽입

sarangdaddy committed:
아마존 로고 이미지 첨부

sarangdaddy committed:
CSS 최초 셋팅

- reset.css
- styles.css

sarangdaddy committed:
header bar에 대한 HTML 수정

- class 명칭 변경
- input & button 을 form 태그 안에 삽입
- 로고 사진 비율 변경

sarangdaddy committed:
header-bar css 업데이트

- 기본 레이아웃 완성

sarangdaddy committed:
header-bar 로그인 레이아웃 추가

- login 항목 밑에 로그인 안내 레이아웃 출력

sarangdaddy committed:
접속 1초 후 로그인 레이아웃 출력

- HTML : section__login__active -> section__login 으로 클래스명 변경
- section__login에 hidden, visible 클래스 추가시 opacaity 0~1로 변경
- section__login 1초 후 출력위해 js 페이지 생성
- section-view.js에 setTimeout 이벤트로 1초 후 클래스명 변경 기능 추가

sarangdaddy committed:
section__login 호버 설정

- header-bar의 로그인 항목에 마우스를 호버하면 section__login이 사라진다.

sarangdaddy committed:
section__login2 레이아웃 생성

- header의 로그인 영역을 호버하면 section__login1이 사라지고 section__login2가 나온다.
- section__login2에는 detail항목이 추가되어 있다.
- detail항목들은 HTML UL로 우선 구현했다.

sarangdaddy committed:
README UPDATE

sarangdaddy committed:
CSS 코드 수정

- flex : 1 -> flex-grow 정확한 명칭 사용

sarangdaddy committed:
login_section 사라지는 기능 추가

- class이름 visible과 hidden을 만듬
- 화면에 출력해야 할때는 visible 클래스를 추가.
- 화면에서 사라지게 해야 할때는 hidden 클래스를 추가.
- visible에는 opacity 0 -> 1을 트랜지션 주어서 이쁘게 나오게함.

sarangdaddy committed:
3/8 README 정리

- 상단 네비게이션바 딤처리, 모달 마무리 남음

sarangdaddy committed:
login__section2 CSS 호버 삭제 ->  JS로 등록

- mouseover, mouseout 으로 호버기능 생성
- header__login에서 벗어나면 login__section__Deatail이 사라짐을 방지하기 위해

sarangdaddy committed:
딤처리 기능 추가

- dim 효과를 가진 div생성 (최초 hidden) 상태
- 위 div에 딤 효과 CSS 부여
- hover 이벤트에 딤처리 이벤트 추가

sarangdaddy committed:
section_login 레이아웃 위치 수정

- 단순 위치 수정 커밋

sarangdaddy committed:
배송처 영역 호버 기능 추가 완료

- 배송처 영역 호버시 [주소 변경 레이어] 출력
- HTML section__location 추가
- section__location CSS 파일 생성 및 스타일 적용
- style.css에 section__location.css import 완료

sarangdaddy committed:
section 화살표 수정

- login과 location sction에서 화살표 위치가 다름
- section__arrow에서 __login, __location으로 분류
- 각 section에 맞는 위치로 스타일 수정

sarangdaddy committed:
section__location 딤처리 기능 추가 완료

- JS 이벤트로 딤효과 설정
- section__login은 호버효과도 이벤트로 처리했는데
- section__location은 호버효과를 css로 처리함
- 나중에 어떤 차이점과 문제가 발생할지 지켜보도록 함.

sarangdaddy committed:
1주차 계획 50% 완성

- README UPDATE

crongro commented:
커밋단위,커밋로그가 아주 좋군요. 잘했습니다.

crongro commented on css/components/header-bar.css:
```diff
@@ -0,0 +1,85 @@
+.home__header__country,
+.home__header__login,
+.home__header__checkOrder,
+.home__header__basket {
```
여러가지 스타일에 각각 부여하지 않고 이렇게 한번에 적용한 시도 좋아요.
다른방법은 예를들어 
.pad-15 {} 
라는 스타일을 만드는 것이고요.

crongro commented on css/components/section-location.css:
```diff
@@ -0,0 +1,67 @@
+  gap: 15px;
+  background-color: white;
+  color: black;
+  z-index: 4;
```
z index에 대해서 학습할 주제가 하나 있는데 stacking context라는 개념입니다.
쉽게말하면 자식이 아무리 z값이높아도..부모의 낮으면 올라갈 수 없는 이유…

sarangdaddy replied:
z-index 개념을 정리하지 못하고 우선 사용했는데 학습 방향성 알려주셔서 감사합니다!!!!

crongro commented on css/components/section-location.css:
```diff
@@ -0,0 +1,67 @@
+
+.section__arrow__location {
+  position: absolute;
+  width: 18.93px;
```
요거를 absolute를 안쓰면 어떻게 할 수 있을까요?

crongro commented on css/components/section-login.css:
```diff
@@ -0,0 +1,103 @@
+  margin-bottom: 15px;
+}
+
+.section__login.hidden {
```
.section__login 을 제거하면 공통 hidden 클래스가 되는것 같은데요?

crongro commented on index.html:
```diff
@@ -0,0 +1,111 @@
+            </div>
+            <div class="section__login__screen">
+              <ul>
+                <li class="section__loing__liTitle">계정</li>
```
클래스 이름이 언더스코어가 두 개씩인데.. 모두다 그러면 그냥 한개로 안될까요?

crongro reviewed (approved):
기본을 잘 익히시려고 노력한 모습이 보여서 좋네요.
그리고 리드미 정리는 너무 잘하셨고요. 내용이  너무 많으면 다른 마크다운문서나 wiki페이지 작성도 생각해보세요.
